transaction-service/src/test/kotlin/ua/com/foxminded/restClient/kafka/KafkaConsumerServiceTest.kt

```diff
@@ -0,0 +1,24 @@
+package ua.com.foxminded.restClient.kafka
+
+import org.junit.jupiter.api.Test
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.test.autoconfigure.web.reactive.AutoConfigureWebTestClient
+import org.springframework.boot.test.context.SpringBootTest
+import org.springframework.kafka.core.reactive.ReactiveKafkaProducerTemplate
+import proto.ProtoMessage
+import utils.Transactions
+
+@SpringBootTest
+@AutoConfigureWebTestClient
+class KafkaConsumerServiceTest {
+
+
+    @Autowired
+    private lateinit var kafkaProducerTemplate: ReactiveKafkaProducerTemplate<String, ByteArray>
+
+    @Test
+    fun `findTransactionsByPersonIdAndTime should return ListResponse when input correct`() {
```

Comment:
if this test is unfinished, leave some disclaimer in it please, because otherwise it looks strange

Reply:
ok